src/VideoToolbox/VTProfessionalVideoWorkflow.cs

```diff
@@ -0,0 +1,36 @@
+// 
+// VTProfessionalVideoWorkflow.cs: VideoTools Professional Video Workflow 
+//
+// Authors: 
+//    Israel Soto (issoto@microsoft.com)
+// 
+// Copyright 2022 Microsoft Corporation.
+//
+
+#if __MACOS__
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+
+using ObjCRuntime;
+using Foundation;
+
+namespace VideoToolbox {
+
+#if NET
+	[SupportedOSPlatform ("macos10.9")]
```

Comment:
Perhaps unavailability attributes for other platforms would be good here in case the #if MONOMAC is ever changed